### PR TITLE
DEP removing the `notebooks`directory

### PR DESCRIPTION
**Pull request by BorisMuzellec**
#### What does your PR implement? Be specific.

This PR removes the `notebooks` directory and replaces references to it with links to the example gallery in the docs.
Indeed, users discovering pydeseq2 should instead refer to the [Getting Started](https://pydeseq2.readthedocs.io/en/latest/auto_examples/index.html) examples from the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@
 
 ## Overview
 
-This package is a python implementation of the [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) method [1]
-for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
+This package is a python implementation of the [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) 
+method [1] for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
```
Comment (maikia) [inline README.md:22]: not a scope of this PR but would be nice if there was a link to [1] leading to the references in the bottom of the page

File: README.md [hunk 2]
```diff
@@ -62,26 +63,31 @@ Please don't hesitate to open an issue in case you encounter any issue due to po
...
 ## Getting started
 
-The [`notebooks` directory](https://github.com/owkin/PyDESeq2/blob/main/notebooks/README.md) contains minimal examples on how to use PyDESeq2, in the form of jupyter notebooks.
+The [Getting Started](https://pydeseq2.readthedocs.io/en/latest/auto_examples/index.html) section of the documentation contains downloadable examples on how to use PyDESeq2.
 
-You can also try them from your browser (on synthetic data only):  
+You can run them directly in your browser:  
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/owkin/PyDESeq2/HEAD?labpath=notebooks%2Findex.ipynb) 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Owkin/PyDESeq2/5bf8ec61831abda43ab472517bf62c1d176b5043?urlpath=lab/tree/jupyter_notebooks/auto_examples/plot_minimal_pydeseq2_pipeline.ipynb)
```
Comment (maikia) [inline README.md:70]: I would remove it unless you are 100% sure this link won't change with new versions? I would guess that it does

**Review by maikia — COMMENTED** [review]
LGTM, just unsure about the binder link

File: README.md (hunk 2 above)
Comment (BorisMuzellec) [reply README.md:70]: Indeed, good point. Let me remove it.

**Pull request merged**